src/sentry/integrations/bitbucket/repository.py

```diff
@@ -126,3 +126,6 @@ def compare_commits(self, repo, start_sha, end_sha):
                 installation.raise_error(e)
             else:
                 return self._format_commits(repo, res)
+
+    def repository_external_slug(self, repo):
+        return repo.name
```

Comment:
Bitbucket does in fact have a repository id that could be utilized. This could perhaps be done in a different pr if we wanted to change it. We don't have to, however, it'll work as is.

Reply:
We'd also have to update the repository add process for bitbucket to use the numeric id then. These 'slugs' have to match the `identifier` used in the integration repository list API. Its a pretty fragile contract right now, I'd love to revisit and make it harder to accidentally break. This at least moves us away from using fields named `name` for identity comparison.